docs/src/arch-monitoring.md

```
@@ -0,0 +1,10 @@
+# Monitoring
+
+Set `.global.enableMonitoring` to `true` in the [values.yaml](https://github.com/truefoundry/KubeElasti/blob/main/charts/elasti/values.yaml) file to enable monitoring.
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)